Ds3/Helpers/Transferrers/WriteTransferrer.cs

```diff
@@ -43,7 +37,26 @@ internal class WriteTransferrer : ITransferrer
                 request.WithMetadata(metadataAccess.GetMetadataValue(objectName));
             }
 
-            client.PutObject(request);
+            try
+            {
+                client.PutObject(request);
+            }
+            catch (Exception ex)
+            {
+                if (retransmitRetries == 0)
+                {
+                    throw new Ds3NoMoreRetransmitException(
```

Comment:
This should use the same exception that the Get retry code uses.  If need be, update the exception that the Get code throws to match this exception.